### ISSUE TITLE
About IoU on semanticKITTI testing dataset

### ISSUE THREAD

**Issue by liamlin5566** [xieKKKi/MotionBEV#1]
Hi, thanks for sharing your great work.
I noticed that your testing result on semanticKITTI leaderboard is 75.8,  which is higher than the result on the paper. Do you use KITTI-Road dataset for training, or use other hyperparameter setting to achieve the result? BTW, do you provide the pretrained models of 75.8? Is "MotionBEV-kitti-road-test-74.88.pt" the file name of the  pretrained weight of 75.8?
Thanks.

**Comment by xieKKKi** [comment]
> Hi, thanks for sharing your great work. I noticed that your testing result on semanticKITTI leaderboard is 75.8, which is higher than the result on the paper. Do you use KITTI-Road dataset for training, or use other hyperparameter setting to achieve the result? BTW, do you provide the pretrained models of 75.8? Is "MotionBEV-kitti-road-test-74.88.pt" the file name of the pretrained weight of 75.8? Thanks.

Hi, @liamlin5566 . Thank you for your interest in our work! The result on the semanticKITTI leaderboard (IoU 75.8%) was achieved by using KITTI road data for training and combining predictions from a semantic segmentation network to remove immovable objects. The "MotionBEV-kitti-road-test-74.88.pt" model is trained on both SemanticKITTI and KITTI road data, achieving an IoU of 74.88% without relying on semantic information. The results reported in our paper do not use KITTI road data because the hidden SemanticKITTI test sequence 20 fully overlaps with the KITTI road sequence 41 used for validation.

**Comment by liamlin5566** [comment]
Thanks for your reply. I also notice that you will drop some static frames in MotionBEV-semantickitti.yaml, and I wonder whether  "MotionBEV-kitti-road-test-74.88.pt" is also trained with dropping static frames.
Thanks.

**Comment by xieKKKi** [comment]
> Thanks for your reply. I also notice that you will drop some static frames in MotionBEV-semantickitti.yaml, and I wonder whether "MotionBEV-kitti-road-test-74.88.pt" is also trained with dropping static frames. Thanks.

We do drop static frames when training with SemanticKITTI data, but do not drop static frames when training with additional  KITTI road data.

**Comment by liamlin5566** [comment]
OK, thanks for answering my questions?

**Comment by liamlin5566** [comment]
Hi @xieKKKi , sorry for disturbing again.  I ecounter unstable val performance when trained with additional KITTI-Road data and dropping the static frames in original semanticKITTI dataset. In https://github.com/xieKKKi/MotionBEV/issues/1#issuecomment-1855659440, do you means that you do not drop the frames in original semanticKITTI dataset? 
Thaks

**Comment by xieKKKi** [comment]
I set "drop_few_static_frames" in "MotionBEV-semantickitti.yaml" as True when training with SemanticKITTI data, and set it as False when training with both SemanticKITTI data and additional KITTI road data. Can you provide more details about the unstable val performance? Did you train from scratch?

**Comment by xieKKKi** [comment]
> Hi @xieKKKi , sorry for disturbing again. I ecounter unstable val performance when trained with additional KITTI-Road data and dropping the static frames in original semanticKITTI dataset. In [#1 (comment)](https://github.com/xieKKKi/MotionBEV/issues/1#issuecomment-1855659440), do you means that you do not drop the frames in original semanticKITTI dataset? Thaks

I set "drop_few_static_frames" in "MotionBEV-semantickitti.yaml" as True when training with SemanticKITTI data, and set it as False when training with both SemanticKITTI data and additional KITTI road data. Can you provide more details about the unstable val performance? Did you train from scratch?

**Comment by liamlin5566** [comment]
Thank you for replying. I train from scratch. The val moving IoU can reach ~73% while the next epoch drops to ~65% although the change of training loss is small.  I would try to train the model without dropping any frames.

**Comment by liamlin5566** [comment]
Sorry for late replying. The val performance is more stable without dropping any frames.  And I also noticed that some stopped object is marked as moving (https://github.com/PRBonn/semantic-kitti-api/issues/80), which may causing some performance instability in the early training epoch. Thanks.
